src/util.js

```diff
@@ -28,7 +28,7 @@ function varlinkCallChannel(channel, method, parameters) {
 
             // FIXME: support answer in multiple chunks until null byte
             if (data[data.length - 1] != 0) {
-                reject("protocol error: expecting terminating 0");
+                reject(new Error("protocol error: expecting terminating 0"));
```

Comment:
Note: throwing proper Error classes, and adding missing semicolons etc. are unrelated changes, and should be in separate commits (or even PRs). It's okay for now, but for the future.

Reply:
Ok, good to know :)